src/toil/test/cwl/cwlTest.py

```diff
@@ -121,7 +125,7 @@ def path_without_rev():
 
     @slow
     @pytest.mark.timeout(900)
```

Comment:
I think the conformance tests just already longer now. So this timeout will need to be increased